lib/fun_facts.rb

```diff
@@ -0,0 +1,18 @@
+class FunFacts
+
+    def initialize
+    end
+
+    def most_popular_make(api)
```

Comment:
You're on the right track here! Instead of this reading in api data though, you'd want to send it the array of Vehicle objects you created from the api data in Iteration 2, that way you're dealing with Ruby objects, not nested hashes.